index.html

```diff
@@ -14,262 +13,432 @@
     />
   </head>
   <body>
-    <main>
-      <section class="boss">
-          <div class="raj">
-            <header>
-              <div><a href="#" class="logo">My Logo</a></div>
-              <nav class="navbar">
-                <img src="image/Union.png" alt="button" />
-              </nav>
-            </header>
-            <aside class="glad">
-              <h1>I'm Vivek Raj glad to see you !</h1>
-              <p class="software">
-                I’m a software developer! I can help you build a product, feature or
-                website Look through some of my work and experience! If you like what
-                you see and have a project you need coded, don’t hestiate to contact
-                me.
+    <main class="boss">
+      <div class="raj"></div>
+      <header class="head">
+        <div class="logo">My logo</div>
+        <nav class="navbar1">
+          <img src="image/Union.png" alt="button" />
+        </nav>
+        <nav class="navbar">
+          <ul class="container">
+            <li><a href="#facebook">Portfolio</a></li>
+            <li><a href="#About">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </header>
+      <section class="master">
+        <div class="intro">
+          <img class="top-ground" src="image/Header bg@2x.png" alt="back">
+          <aside class="material">
+            <h1 class="see">
+              I’m Vivek Raj.<br />
+              Glad to see you!
+            </h1>
+            <p class="software">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
```

Comment:
- [ ] **[OPTIONAL]** Great job on your portfolio! I just wanted to share a friendly suggestion that could make your project even better. Have you considered increasing the font size of your text on the page? Doing so might bring it closer to the Figma design and make it look even more appealing. Keep up the good work!

   ### Your design:
   ![image](https://github.com/VivekRaaZ/portfolio/assets/107952724/8c00dbdb-511f-4ba0-af19-b448d06d1ec3)

   ### Figma design:
   ![image](https://github.com/VivekRaaZ/portfolio/assets/107952724/733e0fa0-3c61-401e-a972-eccb92140d37)